server/src/server.ts

```diff
@@ -1119,7 +1119,7 @@ function onMessage(msg: p.Message) {
           renameProvider: { prepareProvider: true },
           documentSymbolProvider: true,
           completionProvider: {
-            triggerCharacters: [".", ">", "@", "~", '"', "="],
+            triggerCharacters: [".", ">", "@", "~", '"', "=", "("],
```

Comment:
I added this more as a test. I think we can leave it on in the pre-release version and see what people say. Easy to remove if it's annoying because it triggers completion quite often.